### PR TITLE
Disable tycho consumer poms

### PR DESCRIPTION
**Pull request by nagyo**
See additional details here: https://github.com/eclipse-tycho/tycho/issues/664

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -67,6 +67,7 @@
 				<configuration>
 					<strictBinIncludes>false</strictBinIncludes>
 					<strictVersions>false</strictVersions>
+					<skipPomGeneration>true</skipPomGeneration>
```
Comment (apeteri) [inline pom.xml]: I'm not sure if this one is needed if the `pluginManagement` section below defines it as well. Can you check what the effective POM looks like without this change?

**Review by apeteri — APPROVED** [review]
LGTM with a question below.

File: pom.xml (hunk 1 above)
Comment (nagyo) [reply pom.xml]: You were right, the second one is enough 👍🏻

**Review by cmark — APPROVED** [review]
🥓

**Pull request merged**